client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -741,6 +741,22 @@ Undocumented.prototype.siteConnections = function( siteId, fn ) {
 };
 
 /**
+ * Returns a single site connection.
+ *
+ * @param {Number|String} siteId       The site ID or domain
+ * @param {Number}        connectionId The connection ID to get.
+ * @api public
+ * @return {Promise} A Promise to resolve when complete.
+ */
+Undocumented.prototype.getSiteConnection = function( siteId, connectionId ) {
```

Comment:
Should this be added to `lib/wpcom-undocumented/lib/site.js` as `getConnection` so it can be called as such?

```js
wpcom.undocumented().site( siteId ).getConnection( connectionId )
```